<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(10, 15, 30);</string>
  </property>
  <widget class="QStackedWidget" name="Windows">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>290</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="main_win">
    <widget class="QPushButton" name="start_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>290</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#start_btn{
	background-color: rgb(33, 133, 179);
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

QPushButton#start_btn:hover{
	background-color: rgb(144, 65, 255);
	color: rgb(255, 255, 255);
}

QPushButton#start_btn:pressed{
	background-color: rgb(255, 255, 255);
	color: rgb(10, 15, 30);
}</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>290</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#stop_btn{
	background-color: rgb(203, 0, 27);
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

QPushButton#stop_btn:hover{
	
	background-color: rgb(113, 8, 62);
	color: rgb(255, 255, 255);
}

QPushButton#stop_btn:pressed{
	background-color: rgb(255, 255, 255);
	color: rgb(10, 15, 30);
}</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_main">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>360</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>DON'T BE AFK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="made_by_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>360</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>MADE BY EVILSPAZE©</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="settings_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>dripicons-v2</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#settings_btn{
	color:rgb(255, 255, 255);
	border-radius:1px;
}

QPushButton#settings_btn:hover{
	color:rgb(33, 133, 179);
}</string>
     </property>
     <property name="text">
      <string>~</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="info">
    <widget class="QPushButton" name="back_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>dripicons-v2</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#back_btn{
	color:rgb(255, 255, 255);
	border-radius:1px;
}

QPushButton#back_btn:hover{
	color:rgb(33, 133, 179);
}</string>
     </property>
     <property name="text">
      <string>q</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="start_key_bind_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#start_key_bind_btn{
	
	color: rgb(10, 15, 30);
	background-color: rgb(255, 255, 255);
	border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
}

QPushButton#start_key_bind_btn:hover{
	color:rgb(33, 133, 179);
}</string>
     </property>
     <property name="text">
      <string>F8</string>
     </property>
    </widget>
    <widget class="QLabel" name="start_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(33, 133, 179);
color: rgb(255, 255, 255);

border-top-left-radius: 10px;
border-bottom-left-radius: 10px;</string>
     </property>
     <property name="text">
      <string>start AFK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="stop_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(203, 0, 27);
color: rgb(255, 255, 255);

border-top-left-radius: 10px;
border-bottom-left-radius: 10px;</string>
     </property>
     <property name="text">
      <string>stop AFK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_key_bind_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#stop_key_bind_btn{
	
	color:rgb(10, 15, 30);
	background-color: rgb(255, 255, 255);
	border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
}

QPushButton#stop_key_bind_btn:hover{
	color:rgb(203, 0, 27);
}</string>
     </property>
     <property name="text">
      <string>F8</string>
     </property>
    </widget>
    <widget class="QLabel" name="made_by_label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>360</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>MADE BY EVILSPAZE©</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="error_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>360</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(203, 0, 27);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="guide_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>360</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Don't press on multiple keys while you are picking keybind</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="title_settings">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>360</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>SETTINGS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="on_press_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>360</width>
       <height>290</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 150);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>PRESS ON A BUTTON</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>back_btn</zorder>
    <zorder>start_label</zorder>
    <zorder>stop_label</zorder>
    <zorder>stop_key_bind_btn</zorder>
    <zorder>start_key_bind_btn</zorder>
    <zorder>made_by_label_2</zorder>
    <zorder>error_label</zorder>
    <zorder>guide_label</zorder>
    <zorder>title_settings</zorder>
    <zorder>on_press_label</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
